lavalink/events.py

```diff
@@ -11,6 +11,8 @@ class QueueEndEvent(Event):
     player: BasePlayer
         The player that has no more songs in queue.
     """
+    __slots__ = 'player'
```

Comment:
Doesn't `__slots__` need to be a list/tuple?

Reply:
Linter tells me to remove redundant `()`, so I'm gonna guess it doesn't